src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -1,20 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-browser;$(NetCoreAppCurrent)-ios;$(NetCoreAppCurrent)-tvos;$(NetCoreAppCurrent)-android</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum)</TargetFrameworks>
```

Comment:
Why do we need to build this for .NET 8?

Reply:
Where are the problems with building the non-working inbox version via independent project?

Reply:
Hm.  The original plan was to typeforward BinaryFormatter out into a new library and do the package trick there (which would still have required making two copies), but it turns out that System.Runtime.Serialization.Formatters.dll is itself basically the minimum closure of BinaryFormatter.

It sounds like your suggestion is to sort of do that again, making something like

```
src/libraries/System.Runtime.Serialization.Formatters/src/  :  IsPackable, net9
src/libraries/System.Runtime.Serialization.Formatters.Inbox/src/ : !IsPackable, net9, also claim to be System.Runtime.Serialization.Formatters.dll, <Compile Include=../../System.Runtime.Serialization.Formatters/src/AllTheThingsThatArentBinaryFormatter />
```

I don't know how well our build system would like that.  And since the preview snap date is Tuesday, I don't know that I have the time to find out before then.  We could explore it after.

Reply:
Yes, something like that. The current scheme where the build for NetCoreAppMinimum is meant for NetCoreAppCurrent is not easy to understand. It is fine to explore something better later. It is possible that what you have put together is the best out of all bad solution.

Reply:
NuGet limits us here. It doesn't support specifying multiple TFMs that map to the same TargetFrameworkMoniker+TargetPlatformMoniker tuple in a project. There's a proposal up to fix that eventually but until then, separate projects need to be used in such scenarios.

We have multiple source generator project files per rdifferent roslyn targeting pack version because of the exact same reason.